### DISCUSSION TITLE
Do thumbnails under Cameras update automatically?

### DISCUSSION THREAD

**Discussion by jshacar** [blakeblackshear/frigate#6421] [Ask A Question]
I've just installed version 0.12.0 via Docker Compose on an Ubuntu Server 22.04 VM. The thumbnails on the Cameras page are frozen and only update if I leave the page and return or refresh the page.

I previously had version 0.10.0 installed on another system and a could swear that the camera thumbnails would update every 3-5 seconds. I realize that Birdseye should be used for a higher quality live view. Can provide more environment details if needed.

**Comment by NickM-27** [comment]
No, that page has never had auto updating thumbnails

**Comment by lowlyocean** [comment]
Might be worth documenting it (or if already documented, making it more visible). I migrated from Motioneye and definitely expected that the homepage (Cameras) would be a live view of all detection streams

**Reply by NickM-27** [reply]
This is how it works in the next version of frigate https://github.com/blakeblackshear/frigate/discussions/11136
